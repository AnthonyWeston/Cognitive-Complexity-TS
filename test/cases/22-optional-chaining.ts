const o = { attr: "attr" };

function optionalChaining() {
    o?.attr; // no increment for short-hands
}
