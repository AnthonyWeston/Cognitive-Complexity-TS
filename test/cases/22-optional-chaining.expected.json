{
    "22-optional-chaining.ts": {
        "score": 0,
        "inner": [
            {
                "name": "optionalChaining",
                "score": 0,
                "line": 3,
                "column": 1
            }
        ]
    }
}